renku/core/commands/workflow.py

```diff
@@ -20,10 +20,11 @@
 
 from datetime import datetime
 from pathlib import Path
-from typing import List, Optional
+from typing import Any, Dict, List, Optional
 
 from renku.core import errors
 from renku.core.commands.format.workflow import WORKFLOW_FORMATS
+from renku.core.commands.update import execute_workflow
```

Comment:
Should we move `execute_workflow` function to `renku/core/commands/workflow.py`?